flake.nix

```diff
@@ -4,7 +4,7 @@
   nixConfig.extra-substituters = [ "https://nixos-search.cachix.org" ];
   nixConfig.extra-trusted-public-keys = [ "nixos-search.cachix.org-1:1HV3YF8az4fywnH+pAd+CXFEdpTXtv9WpoivPi+H70o=" ];
 
-  inputs.nixpkgs.url = "nixpkgs/nixos-unstable";
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs?rev=e3652e0735fbec227f342712f180f4f21f0594f2";
```

Comment:
I don't think this should be here

Reply:
You're probably right, I think this was a hacky workaround to make things run on my local machine.

I've mostly moved on from nix development, so someone else will need to pick up this PR (also I still don't know elm).

Reply:
sure